timemachine/md/minimizer.py

```diff
@@ -578,11 +581,16 @@ def local_minimize(
     assert_energy_decreased: bool
         Throw an assertion if the energy does not decrease
 
-    restraint_k float, optional
+    restraint_k: float, optional
         Restraint k to wrap val_and_grad_fn in a positional harmonic restraint to the input positions of the local_idxs.
         If None, minimize with no positional restraint. Refer to `timemachine.potentials.bonded.harmonic_positional_restraint`
         for implementation.
 
+    restrained_idxs: np.ndarray, optional
+        A subset of idxs to restrain, must be a subset of local_idxs. If not provided and
```

Comment:
I'd simplify this (and the double negatives) 

If None then will be set to `local_idxs`.

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1421/commits/20c2ee2b5363a5fe893bce3da4f1db3436a41a77